From 3c5e5dd6660f302cef64e69f232f51491848a59d Mon Sep 17 00:00:00 2001
From: Dimitar Dimitrov <dimitar@dinux.eu>
Date: Thu, 10 Nov 2016 22:03:45 +0200
Subject: [PATCH 09/13] testsuite: Workaround stack underflow with PRU

TODO - should we rely on the DejaGNu workaround that moves the stack?

Signed-off-by: Dimitar Dimitrov <dimitar@dinux.eu>
---
 gcc/testsuite/gcc.c-torture/execute/pr47237.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gcc/testsuite/gcc.c-torture/execute/pr47237.c b/gcc/testsuite/gcc.c-torture/execute/pr47237.c
index 0802531..bfd6c18 100644
--- a/gcc/testsuite/gcc.c-torture/execute/pr47237.c
+++ b/gcc/testsuite/gcc.c-torture/execute/pr47237.c
@@ -1,4 +1,4 @@
-/* { dg-xfail-if "can cause stack underflow" { nios2-*-* } "*" "" } */
+/* { dg-xfail-if "can cause stack underflow" { nios2-*-* pru*-*-*"} "*" "" } */
 /* { dg-require-effective-target untyped_assembly } */
 #define INTEGER_ARG  5
 
-- 
2.10.2

